### PR TITLE
Icons

### PR DESCRIPTION
**Pull request by xuwhite**

### CODE REVIEW

File: resources/views/admin/locations/index.twig [hunk 1]
```diff
@@ -8,7 +8,9 @@
     <div class="container">
         <h1>
             {% if not is_index|default(false) %}
-                {{ m.button(m.icon('chevron-left'), location ? url('/locations', {'action': 'view', 'location_id': location.id}) : url('/admin/locations'), 'secondary', 'sm') }}
+                {{ m.button(m.icon('chevron-left'), location
+                    ? url('/locations', {'action': 'view', 'location_id': location.id})
+                    : url('/admin/locations'), 'secondary', 'sm', __('Back')) }}
```
Comment (MyIgel) [inline resources/views/admin/locations/index.twig:13]: This (and all following) should be translation strings like `general.back`

File: resources/views/pages/design.twig [hunk 2]
```diff
@@ -159,6 +159,68 @@
...
+            <div class="row">
+                <div class="col-md-6 col-lg-4 mb-4">
+                    cancel: {{ m.icon('x-lg') }}<br>
+                    delete: {{ m.icon('trash', 'danger') }}<br>
+                    edit: {{ m.icon('pencil') }}<br>
+                    preview: {{ m.icon('eye','info') }}<br>
+                    reset: {{ m.icon('arrow-counterclockwise') }}<br>
+                    save: {{ m.icon('save', 'primary') }}<br>
+                    <br>
+                    {{ f.button('Cancel', {'icon_left': 'x-lg'}) }}<br>
```
Comment (MyIgel) [inline resources/views/pages/design.twig:174]: You could add a `class="mb-3"` to the br so that it has better spacing

**Review by MyIgel — CHANGES_REQUESTED** [review]
That looks great! <3

Eventually the "delete all sessions" button should have `'title': ' '` set and two smaller things listed below:

File: resources/views/pages/design.twig (hunk 2 above)
Comment (xuwhite) [reply resources/views/pages/design.twig:174]: wont do

**Pull request merged**